io/net/vlan_test.py

```diff
@@ -175,32 +129,29 @@ def test_vlan_1_2230(self):
         self.vlan_port_conf("1", "2230")
         # before ping need few sec for interface to set vlan
         time.sleep(5)
-        if self.ping_check_host(self.host_intf, self.ip_dic[self.peer_intf]):
-            self.fail("Ping test failed for vlan 1 & 2230 in host")
-        self.log.info("Ping test passed for vlan 1 % 2230 in host")
-        if self.ping_check_peer(self.peer_intf, self.ip_dic[self.host_intf]):
-            self.fail("Ping test failed for vlan 1 & 2230 in peer")
-        self.log.info("Ping test passed for vlan 1 % 2230 in peer")
+
+        self.host_interface.ping_check(peer_ip=self.peer_ip, count=5)
+        self.peer_interface.ping_check(peer_ip=self.peer_ip, count=5)
```

Comment:
is this something your doing self ping check?
Or ping from peer to host? if that is the case it should be,
self.peer_interface.ping_check(peer_ip=<host_ip>, count=5)

Reply:
@vaishnavibhat can you please re-check on this and attached the latest logs?
So that we can ask for next steps.